api/src/main/resources/liquibase.xml

```diff
@@ -71,4 +71,12 @@
 
     </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+    <changeSet id="${project.parent.artifactId}-20230323-1212" author="fdeniger">
```

Comment:
It looks like the substitution for `${project.parent.artifactId}` did not happen :)